test/logging/logging.go

```diff
@@ -17,10 +17,9 @@ limitations under the License.
 // logging.go contains the logic to configure and interact with the
 // logging and metrics libraries.
 
-package test
+package logging
```

Comment:
how about testlogging since it's consumed by tests

Reply:
The library is already under test with test/logging. I dont think we should change it to testlogging. Else, it would be test/testlogging which sounds weird.

Reply:
make sense.